tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,31 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _root_locator = (By.CLASS_NAME, 'island')
+
+    _name_locator = (By.CSS_SELECTOR, ".item > .info > h3")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.find_element(
+            *self._name_locator)).is_displayed()
```

Comment:
It's a good practice to find the element outside of the wait, and then just wait for it to be displayed. This avoids the expense of repeating the element lookup:

```python
el = self.find_element(*self._name_locator)
self.wait.until(lambda _: el.is_displayed())
```